assets/css/main.scss

```diff
@@ -41,8 +41,50 @@ img.rounded,
   box-shadow: 0 2px 15px darken($gray, 20%);
 }
 
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: flex-start;
+}
+
+.container_stat {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  align-items: flex-start;
+ }
+ 
+ .stat_big {
+   color: #b0197e;
+   font-size: 2.5em;
+   font-weight: 600;
+   margin: 0%;
+ }
+
+ .stat_tag {
+   margin: 0.2em;
+ }
+ 
+ .stat_card {
+   text-align: center;
+   width: 200px;
+ }
+
+ .stat_desc {
+    font-size: 0.6em;
+    font-weight: 600;
+ }
+
+.quad_image {
+  display:grid;
+  grid-template-columns: 25% 25% 25% 25%;
+}
+
 .card-header {
-  white-space: nowrap;
+  // white-space: nowrap;
```

Comment:
Is this because of the transitions being fucky? If it's causing bugs we can just delete

Reply:
Yes. That caused the bootstrap cards used for blog to become too wide for the page on mobile, but doesn't seem to impact anything else. Will delete.

Reply:
Comment deleted.